### PR TITLE
fix(DB/gameobject): Book of Ur should be lootable by all

### PR DESCRIPTION
**Pull request by Zoidwaffle**
## Changes Proposed:
- Lower respawn rate from 24h to 2s

## Issues Addressed:
- Impossible to finish the quest [The Book of Ur](https://classic.wowhead.com/quest=1013/the-book-of-ur) as a group 

## Tests Performed:
- Applied SQL
- Restarted server

## How to Test the Changes:

`.tele sfk`
Enter SFK and run to the room with Fenrus (the big wolf)
`.q a 1013`
Loot the book
Without the patch, wait 24 hours for it to respawn...
With the patch, others can loot it almost instantly afterwards

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
damn i dunno why so many items are like that o_0
i'm sure we missing more

File: data/sql/updates/pending_db_world/rev_1611782665942053526.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611782665942053526');
+
+-- Lower respawn rate of The Book of Ur
+￼
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1611782665942053526.sql:4]: dafuq O.o

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1611782665942053526.sql:4]: ?

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1611782665942053526.sql:4]: You see the little weird character on L#4

Comment (BarbzYHOOL) [reply data/sql/updates/pending_db_world/rev_1611782665942053526.sql:4]: i don't see it but there seems to be one when i edit the file o_0

Comment (Zoidwaffle) [reply data/sql/updates/pending_db_world/rev_1611782665942053526.sql]: Could you check again? I don't see it... but I tried to fix the weird empty line...

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

**Pull request merged**